changelog.d/13346.misc

```diff
@@ -0,0 +1 @@
+Fix `get_pdu` asking every remote destination even after it finds an event.
```

Comment:
Do we know when the bug was introduced? Or if its been there forever then saying something like:

```suggestion
Fix long-standing bug where Synapse sometimes asked many different remote hosts for the same event.
```